src/main/java/androidx/graphics/Path.kt

```diff
@@ -116,3 +116,18 @@ inline infix fun Path.xor(p: Path): Path {
         op(p, Path.Op.XOR)
     }
 }
+
+/**
+ * Create a path with closed shape from [points].
+ */
+inline fun createPathFromPoints(points: List<PointF>): Path {
```

Comment:
This method only creates lines, this is of rather limited use especially for a Path. Canvas has a drawLines() method for instance that can take an array of x/y coordinates to draw the path.

Reply:
drawLines() does not draw a closed shape

Reply:
All you have to do is add an extra line at the end (it's all that Path does).